packages/core/src/components/field-label/field-label.tsx

```diff
@@ -29,6 +29,11 @@ export class FormFieldLabel implements IxComponent {
    */
   @Prop({ reflect: true, mutable: true }) required?: boolean;
 
+  /**
+   * Is the field component invalid
+   */
+  @Prop() isInvalid: boolean = false;
+
```

Comment:
This should be implemented via Prop because label can be used external, use htmlFor or the controlRef to get the validation state